assets/app/components/file-explorer/component.js

```diff
@@ -103,26 +100,25 @@ export default Ember.Component.extend({
     return (path);
   },
 
-  pathToString() {
-
-    var data = this.get('history');
-    var offset = this.get('history_offset');
-    var path = '';
+  pathToString: Ember.computed('history', 'history_offset', function() {
+    let data = this.get('history');
+    let offset = this.get('history_offset');
+    let path = '';
     for (var i = 0; i <= offset; i++) {
       path += data[i] + '\\';
     }
     return (path);
-  },
+  }),
 
   publishSelectedFile() {
 
     let name = this.get('selectedFile').get('name').replace(/\.[^/.]+$/, '');
 
-    let m = this.get('store').createRecord('application', {
+    let m = this.get('store').createRecord('app', {
       alias: name,
       displayName: name,
       collectionName: 'collection',
-      path: this.fullPath(),
+      filePath: this.get('pathToString') + this.get('selectedFile').get('name')
```

Comment:
FYI: You can do `filePath: this.get('pathToString') + this.get('selectedFile.name')